lib/myfinance/entities/sale.rb

```diff
@@ -23,6 +23,7 @@ class Sale < Base
       attribute :id
       attribute :installment_count
       attribute :installment_number
+      attribute :installments
```

Comment:
I think this should be `attribute :installments, Array[Hash]`

Reply:
Agree, all attributes should specify the type, I guess that I was just following the pattern and couldn't see that

Reply:
fixed in 474563e